app/controllers/dashboards_controller.rb

```diff
@@ -3,5 +3,15 @@ def index
     @user = current_user
     @hosting_parties = @user.parties if current_user
     @attending_parties = @user.attending_parties if current_user
+    if !@user.nil?
+      @friends = @user.friends_list
+    end
+  end
+
+  def add_friend
```

Comment:
Nice! For a future refactor, I wonder if we want to move this into a friends controller? We could make it restful with a friends.new perhaps.